### PR TITLE
Update *mm and add fedc

### LINKED ISSUE #26: Add flatpak-external-data-checker for libraries
Per https://github.com/flathub/org.gnome.Gnote/pull/24#issue-1099209201 and https://github.com/flathub/org.pulseaudio.pavucontrol/pull/19 it was rather hairy in January. Maybe it will be less so in the 42 runtime.

### PR DESCRIPTION
**Pull request by wjt**
Fixes #26.

### CODE REVIEW

**Comment by wjt** [comment]
The gtkmm.json is derived from the one at https://github.com/flathub/org.pulseaudio.pavucontrol/pull/19.

**Comment by wjt** [comment]
Includes a bump to the GNOME 42 runtime – one of the modules failed to build without it but I can't remember which one.

File: gtkmm.json [hunk 1]
```diff
@@ -2,14 +2,18 @@
     "name": "gtkmm",
     "sources": [
         {
-            "sha256": "856333de86689f6a81c123f2db15d85db9addc438bc3574c36f15736aeae22e6",
+            "sha256": "0d5efeca9ec64fdd530bb8226c6310ac99549b3dd9604d6e367639791af3d1e0",
             "type": "archive",
-            "url": "https://ftp.gnome.org/pub/GNOME/sources/gtkmm/3.24/gtkmm-3.24.5.tar.xz"
+            "url": "https://download.gnome.org/sources/gtkmm/4.6/gtkmm-4.6.1.tar.xz",
```
Comment (wjt) [inline gtkmm.json:7]: Oh right, this is GTK 4 not 3...

File: gtkmm.json [hunk 2]
```diff
@@ -20,58 +28,51 @@
...
             ]
         },
         {
             "name": "glibmm",
             "sources": [
                 {
-                    "sha256": "69bd6b5327716ca2f511ab580a969fd7bf0cd2c24ce15e1d0e530592d3ff209c",
+                    "sha256": "8008fd8aeddcc867a3f97f113de625f6e96ef98cf7860379813a9c0feffdb520",
                     "type": "archive",
-                    "url": "https://ftp.gnome.org/pub/GNOME/sources/glibmm/2.66/glibmm-2.66.1.tar.xz"
```
Comment (hfiguiere) [inline gtkmm.json:69]: don't upgrade this past 2.66.x. as it is required buy Gtkmm 3.24

File: gtkmm.json [hunk 3]
```diff
@@ -20,58 +28,51 @@
...
-        },
         {
             "name": "sigc++",
             "sources": [
                 {
-                    "sha256": "d082a2ce72c750f66b1a415abe3e852df2eae1e8af53010f4ac2ea261a478832",
+                    "sha256": "8cdcb986e3f0a7c5b4474aa3c833d676e62469509f4899110ddf118f04082651",
                     "type": "archive",
-                    "url": "https://ftp.gnome.org/pub/GNOME/sources/libsigc++/2.10/libsigc++-2.10.7.tar.xz"
+                    "url": "https://download.gnome.org/sources/libsigc++/3.2/libsigc++-3.2.0.tar.xz",
```
Comment (hfiguiere) [inline gtkmm.json:43]: this is an incompatible version. 2.10 is still needed for gtkmm3.24

File: gtkmm.json [hunk 4]
```diff
@@ -107,14 +114,18 @@
             "name": "pangomm",
             "sources": [
                 {
-                    "sha256": "c885013fe61a4c5117fda395770d507563411c63e49f4a3aced4c9efe34d9975",
+                    "sha256": "a27aa77e017b9afce9e751d85bd1cf890abbb3a58bf59d0fac917eef82db3b5b",
                     "type": "archive",
-                    "url": "https://ftp.gnome.org/pub/GNOME/sources/pangomm/2.46/pangomm-2.46.1.tar.xz"
```
Comment (hfiguiere) [inline gtkmm.json:112]: you probablu just want 2.46.2

**Review by hfiguiere — CHANGES_REQUESTED** [review]

**Comment by wjt** [comment]
Thanks as ever for your help, @hfiguiere !

**Pull request merged**